src/rnn.jl

```diff
@@ -635,7 +659,7 @@ function rnntest(r::RNN, ws, x, hx=nothing, cx=nothing;
                  hy = (hx != nothing),
                  cy = (cx != nothing && r.mode == 2),
                  o...)
-    if r.direction == 1; error("rnntest bidirectional not implemented yet"); end
+    #if r.direction == 1; error("rnntest bidirectional not implemented yet"); end
     if r.dropout != 0; error("rnntest dropout not implemented yet"); end
```

Comment:
I think we can add support for dropout fairly easily.  Just have to make sure exactly where it is applied.  Note that it has no effect in a single layer network according to cudnn docs.